service/frontend/workflow_handler_test.go

```diff
@@ -2943,3 +2944,30 @@ func (s *workflowHandlerSuite) TestExecuteMultiOperation() {
 		})
 	})
 }
+
+func (s *workflowHandlerSuite) TestShutdownWorker() {
+	config := s.newConfig()
+	wh := s.getWorkflowHandler(config)
+	ctx := context.Background()
+
+	stickyTaskQueue := "sticky-task-queue"
+
+	expectedMatchingRequest := &matchingservice.ForceUnloadTaskQueueRequest{
+		NamespaceId:   s.testNamespaceID.String(),
+		TaskQueue:     stickyTaskQueue,
+		TaskQueueType: enumspb.TASK_QUEUE_TYPE_WORKFLOW,
+	}
+
+	s.mockNamespaceCache.EXPECT().GetNamespaceID(gomock.Eq(s.testNamespace)).Return(s.testNamespaceID, nil).AnyTimes()
+	s.mockMatchingClient.EXPECT().ForceUnloadTaskQueue(gomock.Any(), gomock.Eq(expectedMatchingRequest)).Return(&matchingservice.ForceUnloadTaskQueueResponse{}, nil)
```

Comment:
I'm assuming there are more comprehensive tests for `ForceUnloadTaskQueue` in the PR that introduced it?

Reply:
Yeah, I can backfill the tests on that.

Reply:
added `TestForceUnloadTaskQueue`